zeena-ahmad/style.css

```diff
@@ -0,0 +1,83 @@
+.sidebar {
+    display: inline-block;
+    height: 100%;
+    width: 220px;
+    background-color: #334856;
+    float: left;
+    padding-top: 100px;
+    padding-bottom:100%;
```

Comment:
using this padding causes to stretch aside more then enough so avoid using that and tray to find another way around it